.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,30 @@
+# Run ShellCheck on PRs and master
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    strategy:
+      matrix:
+        os: ["ubuntu", "macos"]
+    #uses: openedx/.github/.github/workflows/shellcheck.yml@master
+    uses: kdmccormick/openedx.github/.github/workflows/shellcheck.yml@kdmccormick/shellcheck-impl
+    with:
+      # For details on the meaning of each of these arguments, see:
+      # https://githu.com/openedx/.github/blob/master/.github/workflows/shellcheck.yml
+      root-dirs: "./scripts ./.github ./lms ./cms ./xmodule ./common/ ./openedx"
```

Comment:
Why not just `.`?  Are there directories that you're avoiding?

Reply:
I'm avoiding `node_modules`, which isn't a problem in CI, but is present locally.

On second thought, it'd probably be better to include everything, and then allow certain directories to be excluded. I'll make that change now.